File: src/main/java/checkdigitcli/command/Help.java
package checkdigitcli.command;

import picocli.CommandLine.Command;

@Command(name = "help")
public class Help implements Runnable {

	@Override
	public void run() {
		System.out.println("== CheckDigitCLI ==");
		System.out.println("Copyright (c) GenElectrovise 2022");
		System.out.println("https://github.com/GenElectrovise/CheckDigitCLI");
	}
}

File: src/main/java/checkdigitcli/command/Help.kt
package checkdigitcli.command

import picocli.CommandLine

@CommandLine.Command(name = "help")
class Help : Runnable {
    override fun run() {
        println("== CheckDigitCLI ==")
        println("Copyright (c) GenElectrovise 2022")
        println("https://github.com/GenElectrovise/CheckDigitCLI")
    }
}